build/kokoro/build.sh

```diff
@@ -85,72 +87,61 @@ export REPOS_BASE_DIR=${KOKORO_ARTIFACTS_DIR}/github
 build_java () {
   mkdir -p ${REPOS_BASE_DIR}/grpc-java
   git clone https://github.com/stanley-cheung/grpc-java ${REPOS_BASE_DIR}/grpc-java
+  LANG='java'
+  docker_image_tag
   (cd ${REPOS_BASE_DIR}/grpc-java && \
     git checkout o11y-testing-${JOB_MODE} && \
     git log -1 --oneline && \
-    ./gradlew installDist -x test -PskipCodegen=true -PskipAndroid=true)
+    ./buildscripts/observability-test/build_docker.sh && \
```

Comment:
Where is `build_docker.sh`? I don't see it in this PR or on the master branch and I don't see it being programmatically generated anywhere? Maybe accidentally not checked in?

Reply:
So this belongs to each individual language's repo.